bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetAutomationHandler.java

```diff
@@ -0,0 +1,427 @@
+
+    private static final SimpleDateFormat formatter = new SimpleDateFormat("ss.SSS");
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = OpenWebNetBindingConstants.AUTOMATION_SUPPORTED_THING_TYPES;
+
+    // moving states
+    public static final int MOVING_STATE_STOPPED = 0;
+    public static final int MOVING_STATE_MOVING_UP = 1;
+    public static final int MOVING_STATE_MOVING_DOWN = 2;
+    public static final int MOVING_STATE_UNKNOWN = -1;
+
+    // calibration states
+    public static final int CALIBRATION_INACTIVE = -1;
+    public static final int CALIBRATION_ACTIVATED = 0;
+    public static final int CALIBRATION_GOING_UP = 1;
+    public static final int CALIBRATION_GOING_DOWN = 2;
+
+    // positions
+    public static final int POSITION_MAX_STEPS = 100;
+    public static final int POSITION_DOWN = 100;
+    public static final int POSITION_UP = 0;
+    public static final int POSITION_UNKNOWN = -1;
+
+    public static final int SHUTTER_RUN_UNDEFINED = -1;
+    private int shutterRun = SHUTTER_RUN_UNDEFINED;
+    private static final String AUTO_CALIBRATION = "AUTO";
+
+    private long startedMovingAt = -1;
+    private int movingState = MOVING_STATE_UNKNOWN;
+    private int positionEstimation = POSITION_UNKNOWN;
+    private @Nullable ScheduledFuture<?> moveSchedule;
+    private int positionRequested = POSITION_UNKNOWN;
+    private int calibrating = CALIBRATION_INACTIVE;
+    private static final int MIN_STEP_TIME_MSEC = 50;
+    private @Nullable Command commandRequestedWhileMoving = null;
+
+    public OpenWebNetAutomationHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        Object shutterRunConfig = getConfig().get(OpenWebNetBindingConstants.CONFIG_PROPERTY_SHUTTER_RUN);
+        try {
+            if (shutterRunConfig == null) {
+                shutterRunConfig = AUTO_CALIBRATION;
+                logger.debug("shutterRun null --> default to AUTO");
+            } else if (shutterRunConfig instanceof String) {
+                if (AUTO_CALIBRATION.equalsIgnoreCase(((String) shutterRunConfig))) {
+                    logger.debug("shutterRun set to AUTO via configuration");
+                    shutterRun = SHUTTER_RUN_UNDEFINED; // reset shutterRun
+                } else { // try to parse int>=1000
+                    int shutterRunInt = Integer.parseInt((String) shutterRunConfig);
+                    if (shutterRunInt < 1000) {
+                        throw new NumberFormatException();
+                    }
+                    shutterRun = shutterRunInt;
+                    logger.debug("shutterRun set to {} via configuration", shutterRun);
+                }
+            } else {
+                throw new NumberFormatException();
+            }
+        } catch (NumberFormatException e) {
+            logger.warn("Wrong configuration: {} setting must be {} or an integer >= 1000",
```

Comment:
It's a user error and also set in the status so it should be log to debug.
```suggestion
            logger.debug("Wrong configuration: {} setting must be {} or an integer >= 1000",
```